src/documents/tutorial-userInterfaceOptions/IntegratingUIOptionsStylingPreferences.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Integrating UI Options Styling Preferences
+category: Tutorials
+---
+
+Enactors modify a website to meet the preferences specified through UI Options. However, not all enactors modify
+the styling of a website. For those that do, stylesheets are provided in `src/framework/preferences/css/` to facilitate
+this application. Each styling related enactor will provide its own stylesheet(s), so that an integrator can import only
+the styles for the enactors they are using in their particular configuration. Alternatively the `Enactors.css` or
+`Enactors_base.css` stylesheets can be used to quickly access all of the available styling enactors styles.
+
+## Sass
+
+The styles for UI Options are written using [Sass](https://sass-lang.com), using
+[SCSS syntax](https://sass-lang.com/documentation/syntax#scss). All of the related Sass files are contained within the
+`src/framework/preferences/css/sass`. These stylesheets are used to style the Preference Editors as well provide the
```

Comment:
Give a pointer into package.json as to which bit of the build actually operates sass and how if needs to be configured (if it needs to be)